partiql-lang/src/main/kotlin/org/partiql/lang/planner/transforms/plan/RexConverter.kt

```diff
@@ -35,28 +42,29 @@ internal object RexConverter : PartiqlAst.VisitorFold<RexConverter.Ctx>() {
     data class Ctx(
         val node: PartiqlAst.PartiqlAstNode,
         var rex: Rex? = null,
+        val problemHandler: ProblemHandler
```

Comment:
FYI I'm relatively newer to this portion of the code base. Whenever an error is encountered in these conversions in the existing code, an error is thrown. Trying to understand why in this PR we're now including the `problemHandler` in `RexConverter` and `RelConverter`. Now there's a mix of problems being added and errors thrown.

Perhaps it should be a separate PR to transition the error throwing to use the `ProblemHandler`? I'm not opposed to using `ProblemHandler`. Just trying to better understand why its being changed in this PR.

Reply:
That's understandable. I'll look into reverting those changes.